java/code/src/com/suse/manager/webui/services/impl/SaltService.java

```diff
@@ -320,8 +327,26 @@ private String callToString(AbstractCall<?> call) {
         return String.format("[%s.%s]", call.getModuleName(), call.getFunctionName());
     }
 
+    private static Map<String, Object> filterPayload(Map<String, Object> payload) {
+        var kwarg = payload.get("kwarg");
+        if (kwarg == null) {
+            return payload;
+        }
+        var kwargFiltered = kwarg instanceof Map ?
+                ((Map<?, ?>) kwarg).entrySet().stream()
+                .filter(e -> e.getValue() != null)
+                .collect(Collectors.toMap(
+                        Entry::getKey,
+                        e -> SENSITIVE_DATA_PATTERN.matcher(e.getValue().toString()).find() ?
+                                "HIDDEN" :
+                                e.getValue())) :
+                kwarg;
```

Comment:
Have to filter very carefully here cause the underlying Map isn't typed. I wonder if it covers all cases, probably can extend the unit tests, but I think it's fine

Reply:
Looks reasonable for now. We can improve later if needed